src/Aquarius.ONE.ClientSDK/ClientSDK.cs

```diff
@@ -91,52 +91,52 @@ public PlatformEnvironment Environment
         private void InstantiateAPIs()
         {
 
-            Authentication = new AuthenticationApi(_environment, ContinueOnCapturedContext);
+            Authentication = new AuthenticationApi(_environment, ContinueOnCapturedContext, ThrowAPIErrors);
             Authentication.Event += Logger.Logger_Event;
 
             _restHelper = new RestHelper(Authentication, Environment, ContinueOnCapturedContext, LogRestfulCalls, ThrowAPIErrors);
             _restHelper.Event += Logger.Logger_Event;
 
-            Core = new CoreApi(Environment, ContinueOnCapturedContext, _restHelper);
+            Core = new CoreApi(Environment, ContinueOnCapturedContext, _restHelper, ThrowAPIErrors);
             Core.Event += Logger.Logger_Event;
 
             UserHelper = new UserHelper(Authentication, Core);
             CacheHelper = new CacheHelper(this);
 
-            Configuration = new ConfigurationApi(Environment, ContinueOnCapturedContext, _restHelper);
+            Configuration = new ConfigurationApi(Environment, ContinueOnCapturedContext, _restHelper, ThrowAPIErrors);
             Configuration.Event += Logger.Logger_Event;
 
             // This is a wrapper around the ConfigurationApi no events are logged directly
             Logbook = new LogbookApi(Environment, ContinueOnCapturedContext, _restHelper);

```

Comment:
LogbookApi creates a ConfigurationApi class so it needs to be able to pass ThrowApiErrors to that class.